src/components/tabs/CdrTabs.vue

```diff
@@ -228,12 +230,11 @@ const style = useCssModule();
           :ref="el => { tabElements[index] = el }"
           :id="getTabId(tab.name)"
           :disabled="tab.disabled"
-          :aria-disabled="tab.disabled"
-          :aria-selected="(selectedIndex === index && !tab.disabled) ? true : false"
-          :tabIndex="(selectedIndex === index && !tab.disabled) ? 0 : -1"
+          :aria-selected="checkIfActive(index, tab) ? true : false"
```

Comment:
Isn't the ternary here redundant? You're already returning a true or false from the method

Reply:
Good catch. updated.